module/zfs/dsl_scan.c

```diff
@@ -1550,6 +1588,18 @@ dsl_scan_sync(dsl_pool_t *dp, dmu_tx_t *tx)
 	if (!scn->scn_async_stalled && !dsl_scan_active(scn))
 		return;
 
+	if (scn->scn_phys.scn_func == POOL_SCAN_REBUILD) {
+		if (scn->scn_visited_this_txg == 19890604) {
```

Comment:
It's just a historical date that happened to pop up. Anything else would work, like the usual deadbeef babyface stuff.